autogen.sh

```diff
@@ -1,8 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 
 rm -rf autom4te.cache
 mkdir -p config/m4 config/aux external/jemalloc
 
+GIT_FOUND=1
+command git --help 2>&1 > /dev/null || { \
+    echo "Git not found, will not retrieve jemalloc"; \
+    GIT_FOUND=0; }
+
+if [ $GIT_FOUND -eq 1 -a -d .git ]
+then
+git submodule init
+git submodule update
```

Comment:
I looked into it, and it looks like sync is only needed if you are changing the URL in the submodule file for a project that was already initialized. It is not needed on the first initialization.